.github/workflows/refapp-2x-appointment.yml

```diff
@@ -0,0 +1,45 @@
+name: RefApp 2.x Appointment
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+    paths:
```

Comment:
@Irenyak1 thanks for the work, i think the paths should be removed since we are now only dealing with 2.x only.
have a look-see at https://github.com/openmrs/openmrs-contrib-qaframework/commit/270e177a1631bf24da0b7390533b48b97c6096a9